From hackermail Thu Jan  1 00:00:00 1970
From: SeongJae Park <sj@kernel.org>
To: Andrew Morton <akpm@linux-foundation.org>
Cc: SeongJae Park <sj@kernel.org>, damon@lists.linux.dev, kernel-team@meta.com, linux-kernel@vger.kernel.org, linux-mm@kvack.org
Message-Id: <20250513002715.40126-4-sj@kernel.org>
In-Reply-To: <20250513002715.40126-1-sj@kernel.org>
Date: Mon, 12 May 2025 17:27:12 -0700
Subject: [PATCH 3/6] mm/damon/paddr: remove unused variable, folio_list, in damon_pa_stat()

Commit c0cb9d91bf297 ("mm/damon/paddr: report filter-passed bytes back
for DAMOS_STAT action") added unused variable in damon_pa_stat(), due to
a copy-and-paste error.  Remove it.

Fixes: c0cb9d91bf297 ("mm/damon/paddr: report filter-passed bytes back for DAMOS_STAT action")
Signed-off-by: SeongJae Park <sj@kernel.org>
Link: https://patch.msgid.link/20250513002715.40126-4-sj@kernel.org
Cc: damon@lists.linux.dev
Cc: kernel-team@meta.com
Cc: linux-kernel@vger.kernel.org
Cc: linux-mm@kvack.org
---
 mm/damon/paddr.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mm/damon/paddr.c b/mm/damon/paddr.c
index 1b70d3f36046..e8464f7e0014 100644
--- a/mm/damon/paddr.c
+++ b/mm/damon/paddr.c
@@ -548,7 +548,6 @@ static unsigned long damon_pa_stat(struct damon_region *r, struct damos *s,
 		unsigned long *sz_filter_passed)
 {
 	unsigned long addr;
-	LIST_HEAD(folio_list);
 	struct folio *folio;
 
 	if (!damon_pa_scheme_has_filter(s))
-- 
2.39.5
